packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -63,6 +69,28 @@ const extractAttachedFilesMetadata = (metadata: Record<string, string>): FileMet
   }
 };
 
+/* @conditional-compile-remove(data-loss-prevention) */
+const convertToUiBlockedMessage = (
+  message: ChatMessageWithStatus,
+  userId: string,
+  isSeen: boolean,
+  isLargeGroup: boolean
+): BlockedMessage => {
+  const messageSenderId = message.sender !== undefined ? toFlatCommunicationIdentifier(message.sender) : userId;
+  return {
+    messageType: 'blocked',
+    createdOn: message.createdOn,
+    content: message.content?.message,
```

Comment:
This content is not the same as message content, maybe find a better name for this like:

warningMessage, alternativeText or anything that can help developer understand that is the message to tell you this is blocked but not the message which was deleted

Reply:
Rename to warningText